### ISSUE TITLE
IE11: SCRIPT65535: Ungültiges aufrufendes Objekt.

### ISSUE THREAD

**Issue by martinoppitz** [taylorhakes/promise-polyfill#31]
```js
function handle(self, deferred) {
    while (self._state === 3) {
      self = self._value;
    }
    if (self._state === 0) { // Exception!!!
      self._deferreds.push(deferred);
      return;
    }
    self._handled = true;
    Promise._immediateFn(function () {
```

**Comment by taylorhakes** [comment]
Thank you for the bug report. This has been fixed in version 6.0.1.

https://github.com/taylorhakes/promise-polyfill/commit/5ab398351e9c35a75b94144b191faf3c5bd6aff4

**Comment by trollkotze** [comment]
It seems the error has been reintroduced.
Using version promise-polyfill@7.10, I had the same error appearing in Internet Explorer 11 and Microsoft Edge 15.
Reverting to promise-polyfill@6.0.1 resolved the issue for me. You might want to look into this once more.
